python/enums/data/persons.csv

```diff
@@ -0,0 +1,11 @@
+id,name,age,isMarried,city,state,country

```

Comment:
Because we're interested in only age groups in this exercise and not locations, can we just remove the city/state/country from the CSV? I agree with you that having enums for those complicates the concept, and the age groups example is nice and simple. It'll also make the code easier to read and not have useless fields lying around.